dotCMS/src/main/java/com/dotcms/workflow/helper/WorkflowHelper.java

```diff
@@ -1782,4 +1783,13 @@ public Contentlet getContentletByIdentifier(final String longIdentifier, final b
             return null;
         }
     }
+
+    /**
+     * Used to call and transform a Contentlet
+     * @param con A Contentlet
+     * @return Transformed Map
+     */
+    public Map<String, Object> contentletToMap(final Contentlet con) {
+        return new ContentletToMapTransformer(con).toMaps().get(0);
```

Comment:
this can (though should not) throw an unchecked indexOutOfBounds exception - I don't see any handling code